index.php

```diff
@@ -95,7 +95,7 @@ private function build_index() {
 
 			$uri = trim( sprintf( 
 					'%s/%s',
-					trim( $uri_info['dirname'], '/' ),
+					trim( $uri_info['dirname'], '/\\' ),
```

Comment:
One `\` should be enough because it [specifies a list of characters that should be trimmed](http://php.net/manual/en/function.trim.php).